docs/user-guide/get-started.md

```diff
@@ -74,65 +80,59 @@ If you use Prettier alongside Stylelint, you should use their [SCSS shared confi
 }
 ```
 
+Other shared configs include:
+
+- [stylelint-config-html](https://www.npmjs.com/package/stylelint-config-html)
+- [stylelint-config-recommended-vue](https://www.npmjs.com/package/stylelint-config-recommended-vue)
```

Comment:
Encourage adoption of shared configs that bundle syntaxes (as the easiest way to get started) by listing known ones here.